### PR TITLE
DOC: Update docs with Docker instructions

### PR DESCRIPTION
**Pull request by ghadjigeorghiou**
- Update the installation docs with instructions on how to setup tiatoolbox docker container.

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -130,3 +130,62 @@ Once you have a copy of the source, you can install it with:
 
 .. _Github repo: https://github.com/tialab/tiatoolbox
 .. _tarball: https://github.com/tialab/tiatoolbox/tarball/master
+
+Using Docker
+==================
+
+If you want to run TIA toolbox in an isolated environment, you can use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . Furthermore, we host different Dockerfiles in our `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_ github repository.
```
Comment (shaneahmed) [inline docs/installation.rst:137]:
```suggestion
If you would like to run TIA toolbox in an isolated environment, you can use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . We host different Dockerfiles in our `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_ github repository. Please report any issues related to docker image in the `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>` repository. 
```

File: docs/installation.rst [hunk 2]
```diff
@@ -130,3 +130,62 @@ Once you have a copy of the source, you can install it with:
...
 .. _tarball: https://github.com/tialab/tiatoolbox/tarball/master
+
+Using Docker
+==================
+
+If you want to run TIA toolbox in an isolated environment, you can use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . Furthermore, we host different Dockerfiles in our `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_ github repository. 
+
+After `installing Docker <https://docs.docker.com/get-docker/>`_ (or Docker Desktop), you can use our TIA toolbox image in 3 different ways.
+
+Pull the image from the Github Container Registry
```
Comment (shaneahmed) [inline docs/installation.rst:141]:
```suggestion
1. Pull the image from the Github Container Registry
```

File: docs/installation.rst [hunk 3]
```diff
@@ -130,3 +130,62 @@ Once you have a copy of the source, you can install it with:
...
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code-block:: console
+
+    $ docker pull ghcr.io/tissueimageanalytics/tiatoolbox:latest
+
+Use the image as a base image in a Dockerfile
```
Comment (shaneahmed) [inline docs/installation.rst:147]:
```suggestion
2. Use the image as a base image in a Dockerfile
```

File: docs/installation.rst (hunk 2 above)
Comment (shaneahmed) [reply docs/installation.rst:141]: May be add a heading as you have for build the image locally

File: docs/installation.rst [hunk 4]
```diff
@@ -130,3 +130,62 @@ Once you have a copy of the source, you can install it with:
...
 .. _Github repo: https://github.com/tialab/tiatoolbox
 .. _tarball: https://github.com/tialab/tiatoolbox/tarball/master
+
+Using Docker
+==================
+
+If you want to run TIA toolbox in an isolated environment, you can use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . Furthermore, we host different Dockerfiles in our `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_ github repository. 
+
+After `installing Docker <https://docs.docker.com/get-docker/>`_ (or Docker Desktop), you can use our TIA toolbox image in 3 different ways.
+
```
Comment (shaneahmed) [inline docs/installation.rst:140]:
```suggestion

Use the pre-built docker image
^^^^^^^^^^^^^^^^^^^^
```

**Review by shaneahmed — CHANGES_REQUESTED** [review]
Please can you address the changes as mentioned.

**Review by DavidBAEpstein — CHANGES_REQUESTED** [review]
Change word order to standard English usage, as made explicit in this diff file
[installation_diff.txt](https://github.com/TissueImageAnalytics/tiatoolbox/files/8097287/installation_diff.txt)
However, I didn't know how to typeset my raw version of installation.rst, so I can not be absolutely certain that the typesetting will be correct. 
.

File: docs/installation.rst [hunk 5]
```diff
@@ -130,3 +130,64 @@ Once you have a copy of the source, you can install it with:
 
 .. _Github repo: https://github.com/tialab/tiatoolbox
 .. _tarball: https://github.com/tialab/tiatoolbox/tarball/master
+
+Using Docker
+==================
+
+If you would like to run TIA toolbox in an isolated environment, you can use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . We host different Dockerfiles in our `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_ github repository. Please report any issues related to docker image in the `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>` repository.
```
Comment (shaneahmed) [inline docs/installation.rst:137]:
```suggestion
To run TIA toolbox in an isolated environment, use our `Docker image <https://github.com/tissueimageanalytics/tiatoolbox-docker/pkgs/container/tiatoolbox>`_ . We host different Dockerfiles in our github repository `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>`_ github repository. Please report any issues related to docker image in the repository `tiatoolbox-docker <https://github.com/TissueImageAnalytics/tiatoolbox-docker>` repository. 
```

**Comment by shaneahmed** [comment]
> Change word order to standard English usage, as made explicit in this diff file [installation_diff.txt](https://github.com/TissueImageAnalytics/tiatoolbox/files/8097287/installation_diff.txt) However, I didn't know how to typeset my raw version of installation.rst, so I can not be absolutely certain that the typesetting will be correct. .

Thanks David. I have added your suggestions above. @ghadjigeorghiou please address these. We can merge once @Srijay-lab  is happy.

**Review by shaneahmed — APPROVED** [review]

**Pull request merged**